### DISCUSSION TITLE
How do we add transitions while collapsing the panels ?

### DISCUSSION THREAD

**Discussion by Harsha1401** [bvaughn/react-resizable-panels#376] [Q&A] [answered]
I need to added some transitions while collapsing/ expanding the panel for better user experience.
Can someone provide examples for this.

**Comment by bvaughn** [comment, marked as answer]
This library doesn't really support animation. You can find a similar discussion thread in #310. Adding transitions would work for toggling expanded/collapsed, but would break drag-to-resize.
